.github/workflows/lint.yml

```
@@ -0,0 +1,22 @@
+name: Lint
+
+on: [push, pull_request]
```

```suggestion
on: [pull_request]
```

The setup on graphene-django is wrong and I need to fix it